store/interfaces.json

```diff
@@ -24,9 +24,17 @@
     "allowed": ["shop-review-summary", "sidebar-close-button"]
   },
   "gallery": {
-    "required": ["product-summary"],
+    "allowed": ["product-summary"],
```

Comment:
This change _could_ cause issues in some stores. I remember a Slack thread about this a few days or weeks back... is this safe now?

Reply:
We discussed it in the last groom. We found out we can define a default block implementation on the `blocks.json` file. That's what I did, so now if there's no explicit gallery definition, we can put a default `product-summary` block there and it will not break :)

Reply:
Awesome!